product/utilities/grid/index.md

```diff
@@ -6,172 +6,90 @@ layout: app
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+		<div class="col-1 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-1</strong></div>
+	</div>
 </div><!--Grid-->
 
 {% highlight html %}
-<div class="Grid">
-	<div class="g-6-12">1</div>
-	<div class="g-6-12">2</div>
-</div><!--Grid-->
-{% endhighlight %}
-
-<h2>12 column responsive modifiers</h2>
-
-<code>__m</code> - $medium-breakpoint modifier
-<code>__l</code> - $large-breakpoint modifier
-
-{% highlight html %}
-<div class="Grid">
-	<div class="g-12-12 g-6-12__m g-1-12__l">1</div>
-	<div class="g-12-12 g-6-12__m g-1-12__l">2</div>
+<div class="container">
+	<div class="row">
+		<div class="col-12 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-12</strong></div>
+		<div class="col-6 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-6</strong></div>
+		<div class="col-6 m-bottom-2 bg-c-g200 t-center p-2"><strong>.col-6</strong></div>
+	</div>
 </div><!--Grid-->
 {% endhighlight %}
 
-<hr />
-
-<h2>Legacy grid</h2>
-
-<p class="t-4">DreamHost legacy floated grid uses <a href="https://neat.bourbon.io/" target="_blank">Bourbon's Neat Sass grid</a> alongside some helper classes from the most common layouts we need to create.</p>
+<hr>
 
-<p class="Alert"><strong>Please note:</strong> Each legacy grid row must have a wrapper with the <code>u-clearfix</code> class attached as it is based on a float model.</p>
-
-<hr />
+<h2>12 column responsive modifiers</h2>
 
-<div class="m-bottom">
-	<div class="u-clearfix m-bottom-2">
-		<div class="bg-c-g200 border-radius p-2 t-center g-1_2"><strong>.g-1_2</strong></div>
-		<div class="bg-c-g200 border-radius p-2 t-center g-1_2"><strong>.g-1_2</strong></div>
-	</div>
-	<div class="u-clearfix m-bottom-2">
-		<div class="bg-c-g200 border-radius p-2 t-center g-1_3"><strong>.g-1_3</strong></div>
-		<div class="bg-c-g200 border-radius p-2 t-center g-2_3"><strong>.g-2_3</strong></div>
-	</div>
-	<div class="u-clearfix m-bottom-2">
-		<div class="bg-c-g200 border-radius p-2 t-center g-1_4"><strong>.g-1_4</strong></div>
-		<div class="bg-c-g200 border-radius p-2 t-center g-3_4"><strong>.g-3_4</strong></div>
+<code>-xs</code> - extra small-breakpoint modifier <576px <br>
```

Comment:
So we actually use different breakpoints than bootstrap's defaults, so you'll want to adjust these values to match what we're using:

md: 750px
lg: 1050px
xl: 1280px